src/styles/ontimize-theme-styles-flat.scss

```diff
@@ -1,171 +1,105 @@
-  $warning: #f1c40f;
-  $warning-hover: #c29d0b;
-  $info: #3498db;
-  $info-hover: #217dbb;
-
-  $default: #bdc3c7;
-  $inverse: #34495e;
-  $disable: #ECF0F1;
+  @include set-color-states($color_states);
 
   .text-muted {
-    color: $default;
+    color: $default-color;
   }
 
   .text-inverse {
     color: white;
   }
 
   .text-primary {
-    color: $primary;
-  }
-
-  a.text-primary:hover {
-    color: #148f77;
+    color: $primary-color;
   }
 
   .text-warning {
-    color: $warning;
-  }
-
-  a.text-warning:hover {
-    color: $warning-hover;
+    color: $warning-color;
   }
 
   .text-danger {
-    color: $danger;
-  }
-
-  a.text-danger:hover {
-    color: $danger-hover;
+    color: $danger-color;
   }
 
   .text-success {
-    color: $success;
-  }
-
-  a.text-success:hover {
-    color: $success-hover;
+    color: $success-color;
   }
 
   .text-info {
-    color: $info;
+    color: $info-color;
   }
 
-  a.text-info:hover {
-    color: $info-hover;
+  a.text-danger:hover,
+  a.text-inverse:hover,
+  a.text-muted:hover,
+  a.text-primary:hover,
+  a.text-success:hover,
+  a.text-warning:hover {
+    opacity: .85;
   }
 
   .o-form .o-form-toolbar .o-form-toolbar-header .mat-toolbar-tools .o-form-toolbar-button.mat-stroked-button,
-  button.mat-button-base:not(.mat-icon-button) {
-    min-width: 150px;
+  button.mat-button-base:not(.mat-icon-button):not(.mat-fab):not(.mat-mini-fab) {
+    min-width: 132px;
   }
 
-  button,
-  button.mat-button-base {
-    border-radius: 0;
-    border: 0;
+  /* Generic button class */
+  @include o-button-class();
 
-    &.o-button-info {
-      color: white;
-      background-color: $info;
-    }
-
-    &.o-button-primary {
-      color: white;
-      background-color: $primary;
-    }
-
-    &.o-button-default {
-      button {
-        color: white;
-        background-color: $default;
-      }
-    }
-
-    &.o-button-success {
-      button {
-        color: white;
-        background-color: $success;
-      }
+  /* MATERIAL BUTTON STYLES*/
+  button,
+  button.mat-button,
+  button.mat-stroked-button {
+    &:not(.mat-icon-button):not(.mat-fab):not(.mat-mini-fab) {
+      border-radius: 0;
     }
 
-    &.o-button-primary {
-      button {
-        color: white;
-        background-color: $primary;
+    &:hover {
+      .mat-button-focus-overlay {
+        opacity: .10;
       }
     }
 
-    &.o-button-danger {
-      button {
-        color: white;
-        background-color: $danger;
-      }
-    }
   }
 
+  /* O-BUTTON STYLES */
   .o-button {
     border: none;
     border-radius: 0;
-    font-weight: normal;
-    -webkit-font-smoothing: subpixel-antialiased;
-    transition: border .25s linear, color .25s linear, background-color .25s linear;
-
-    button {
-      border-radius: 0;
-    }
 
-    &.o-button-info {
+    &.o-button-danger,
+    &.o-button-default,
+    &.o-button-info,
+    &.o-button-primary,
+    &.o-button-success {
       button {
         color: white;
```

Comment:
Could this color 'white' cause problems with dark-mode?

Reply:
It also will be white